lessons/pydata/pandas_types/index.ipynb

```diff
@@ -0,0 +1,6941 @@
+{
```

Comment:
<p>To bychom ale měli spíš nedoporučovat</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PyDataCZ/naucse.python.cz/pull/23/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>

Reply:
<p>Pentagram ti nestačí?</p>